index.html

```diff
@@ -121,6 +130,115 @@ <h2 class="txt-center">
     $('.js-defaultServerIndexApp').on('click', function(){
       app.loadNewServer("http://localhost:18469/api/v1/");
     });
+
+    var appMenu = Menu.buildFromTemplate([
```

Comment:
Does this have to live in index.html? Our other menus live in main.js. If that files is getting too big, I'm sure we could break it up, but IMHO, plopping this in here seems odd.